tests/SonarPartTwoTest.php

```diff
@@ -7,8 +7,18 @@
 
 class SonarPartTwoTest extends TestCase
 {
-    public function  testGivenOneNumberReturnsZero(): void
+    public function  testGivenOneWindowReturnsSix(): void
     {
         $this->assertSame(6, (new SonarPartTwo())->SweepPartTwo("1\n2\n3"));

```

Comment:
you aren't looking for the total you are looking to see if it increases or not, so in this case the answer would be 0

Reply:
Re-written tests.